src/components/progress-tracker/progress-tracker.component.js

```diff
@@ -29,29 +32,39 @@ const ProgressTracker = ({
   showDefaultLabels = false,
   currentProgressLabel,
   maxProgressLabel,
-  orientation = "horizontal",
-  direction = "up",
+  orientation,
+  direction,
```

Comment:
Actually, just to check - removing the default here isn't going to break anything for anyone, right? The default will still be these as far as I can tell from the rest of the code and this is purely to prevent the deprecation warning from triggering on every use of the component? Just wanted to double check.

Reply:
Yeah that's correct

Reply:
Perfect